schedule/__init__.py

```diff
@@ -483,7 +490,8 @@ def run(self):
         """
         logger.info('Running job %s', self)
         ret = self.job_func()
-        self.last_run = datetime.datetime.now()
+        if not isinstance(ret, SkipJob) and ret is not SkipJob:
+            self.last_run = datetime.datetime.now()
         self._schedule_next_run()
```

Comment:
Should `self._schedule_next_run()` also be included in the `if` statements body?